src/modules/card/card.service.ts

```diff
@@ -140,7 +140,7 @@ export class CardService {
         superiorId: Number(area.superiorId) === 0 ? area.id : area.superiorId,
         priorityCode: priority.priorityCode,
         priorityDescription: priority.priorityDescription,
-        cardTypeMethodology: cardType.cardTypeMethodology,
+        cardTypeMethodology: cardType.cardTypeMethodology === stringConstants.M? cardType.cardTypeMethodology:null,
```

Comment:
CardTypeValue should be null also, if the cardType.cardTypeMethodology === "M"  CardTypeValue can be "Safe or UnSafe" otherwise CardTypeValue can be null

Reply:
oh got it